setup.py

```diff
@@ -10,7 +16,7 @@
     author_email='jtharp@objectrocket.com',
     url = 'https://github.com/objectrocket/elasticstat',
     download_url = 'https://github.com/objectrocket/elasticstat/archive/1.3.0.tar.gz',
-    install_requires=['elasticsearch', 'packaging', 'certifi'],
+    install_requires=read_requirements('prod'),
```

Comment:
so I know this functionally the same as what existed before but I think ideally we want prod.txt to have pinned dependencies since technically what runs in tox can potentially deviate from what gets installed by setup.py as officially installation.